beacon-chain/sync/subscriber.go

```diff
@@ -765,3 +766,10 @@ func multiAddr(pid peer.ID, stat *peers.Status) string {
 	}
 	return addrs.String()
 }
+
+func rejectGossipMessage(msg *pubsub.Message) pubsub.ValidationResult {
+	if features.Get().EnableFullSSZDataLogging {
+		log.WithField("rawMessage", msg).Debug("Rejected gossip message with data")
+	}
+	return pubsub.ValidationReject
+}
```

Comment:
Could you add a function level comment? 

What is the intended behavior of this function?